com.reprezen.swagedit/src/com/reprezen/swagedit/preferences/SwaggerValidationPreferences.java

```diff
@@ -31,6 +33,7 @@
         implements IWorkbenchPreferencePage, IPropertyChangeListener {
 
     public SwaggerValidationPreferences() {
+        // GRID is needed because we are not attaching the editor fields directly to FieldEditorParent, but to its child
```

Comment:
In that case it couldn't be a FieldEditorPreference anyway, unless we implemented a ListBoxFieldEditor (or whetever), since there's no existing FieldEditor for that sort of control.